docs/tutorials/before-you-start-a-tutorial.md

```diff
@@ -26,6 +26,8 @@ long-term support (LTS version of the operating system).
     * **TIP**: If you're using a fork of the repo, create a working branch in which to do your tutorials. Create a new branch for each tutorial to prevent a mistake in one from affecting your work in another.
     * The [Postman desktop app](https://www.postman.com/downloads/). Because you run the **To-Do service** on your development system with an `http://localhost` hostname, the web-version of Postman can't perform the exercises.
 
+Instructions for installing these on MacOS can be found [here](macos-installation)
```

Comment:
```suggestion
For information about how to configure MacOS for the To-Do service, visit the [MacOS installation guide](macos-installation).
```

Try to use descriptive link text. Good link text tells you what you'll find when you open the link.

Also, wouldn't this be more convenient for Mac users to see this __before__ the Windows steps?

Maybe something like:

```
## Preparing for the tutorials

The following instructions describe how to prepare for running the tutorials on Windows.
For information about how to prepare MacOS for the tutorials, visit the [MacOS installation guide](macos-installation).

```